src/main/java/org/gridsuite/study/server/service/LoadFlowService.java

```diff
@@ -120,6 +120,12 @@ public void deleteLoadFlowResults(List<UUID> uuids) {
         }
     }
 
+    public void deleteLoadFlowResults() {
+        String path = UriComponentsBuilder
+            .fromPath(DELIMITER + LOADFLOW_API_VERSION + "/results/all").toUriString();
```

Comment:
Path not homogeneous with others.
Remove `all` in th epath or add everywhere

Reply:
Done, I've had to merge two endpoints into one in order to do so. If there's no parameter it deletes all otherwise it deletes what is passed in the parameter